gspread/http_client.py

```diff
@@ -358,6 +359,57 @@ def export(self, file_id: str, format: str = ExportFormat.PDF) -> bytes:
         r = self.request("get", url, params=params)
         return r.content
 
+    def export_worksheet(self, file_id, sheet_id: str, format: str = ExportFormat.PDF) -> bytes:
+        """Export the worksheet in the given format.
+
+        :param str file_id: The key of the spreadsheet to export
+
+        :param str sheet_id: The key of the worksheet to export
+
+        :param str format: The format of the resulting file.
+            Possible values are:
+
+                * ``ExportFormat.PDF``
+                * ``ExportFormat.EXCEL``
+                * ``ExportFormat.CSV``
+                * ``ExportFormat.OPEN_OFFICE_SHEET``
+                * ``ExportFormat.TSV``
+                * ``ExportFormat.ZIPPED_HTML``
+
+            See `ExportFormat`_ in the Drive API.
+
+        :type format: :class:`~gspread.utils.ExportFormat`
+
+        :returns bytes: The content of the exported file.
+
+        .. _ExportFormat: https://developers.google.com/drive/api/guides/ref-export-formats
+        """
+
+        if format not in ExportFormat:
+            raise UnSupportedExportFormat
+
+        if format not in ExportFormat:
+            raise UnSupportedExportFormat
+
+        format_str = "pdf"
+        if format == ExportFormat.PDF:
+            format_str = "pdf"
+        elif format == ExportFormat.CSV:
+            format_str = "csv"
+        elif format == ExportFormat.ZIPPED_HTML:
+            format_str = "zip"
+        elif format == ExportFormat.EXCEL:
+            format_str = "xlsx"
+        elif format == ExportFormat.TSV:
+            format_str = "tsv"
```

Comment:
formats are `StrEnum` objects. so you can use the value of the Enum and it will be in string format.

Reply:
never mind we use the mimetype as value so it won't work.